app/web/features/communities/events/EventPage.tsx

```diff
@@ -130,7 +129,7 @@ function getEventTimeString(
   const start = dayjs(timestamp2Date(startTime));
   const end = dayjs(timestamp2Date(endTime));
 
-  return `${start.format("LLLL")} ${TO} ${end.format(
+  return `${start.format("LLLL")} to ${end.format(
     end.isSame(start, "day") ? "LT" : "LLLL"
   )}`;
 }
```

Comment:
I guess technically unrelated to this namespace, but might be worth getting @papeldeorigami who's worked on the `communities` namespace to follow up on converting this one as this should probably an interpolated string.

Reply:
yes, I didn't want to do it in this PR